src/overlays/actors/ovl_Obj_Jgame_Light/z_obj_jgame_light.h

```diff
@@ -6,10 +6,25 @@
 struct ObjJgameLight;
 
 #define OBJJGAMELIGHT_GET_7F(thisx) ((thisx)->params & 0x7F)
+#define OBJLUPYGAMELIFT_IGNITEFIRE (1 << 0)
+#define OBJLUPYGAMELIFT_DISPLAYCORRECT (1 << 1)
+#define OBJLUPYGAMELIFT_DISPLAYINCORRECT (1 << 2)
```

Comment:
```suggestion
#define OBJLUPYGAMELIFT_IGNITE_FIRE (1 << 0)
#define OBJLUPYGAMELIFT_DISPLAY_CORRECT (1 << 1)
#define OBJLUPYGAMELIFT_DISPLAY_INCORRECT (1 << 2)
```
I thinks its easier to read this way, and matches the style of `SNUFF_FIRE`